aep/0131.yaml

```diff
@@ -0,0 +1,46 @@
+aliases:
+  GetOperation:
+    description: A Get operation is a get on path that ends in a path parameter
+    targets:
+      - formats: ['oas2', 'oas3']
+        given:
+          - $.paths[?(@path.match(/\}']$/))].get
```

Comment:
I think singletons break this rule, since the last element may be a sentinel value: https://aep.dev/156/.

separate from the linter, singletons break the assumptions quite a bit, so I wonder if there's a better way to approach singletons entirely....

Reply:
_If_ I were able to construct a regex that matched paths where the final two segments were static, it would be possible to add that as a `given` here (and exclude it from `ListOperation` alias in 0132). Will this resolve the issue or is it more complicated than that?

Reply:
yes! I think adding it here, as well as excluding it from list, would do the trick.

Reply:
On re-reading aep-156 I see that the URL pattern for singletons is actually: the resource name of the parent followed by one static segment that is a singular noun. This will be difficult to distinguish (using JSON Path) from a collection URL -- the only difference being whether the final segment is singular or plural.

Reply:
And now I see that even this is not enough, since the final segment _could_ be plural, e.g. "configs", for a singleton that supports the "List" operation.  Ugh.

Reply:
I agree this will be quite difficult... one approach I could imagine is that we look at the response schema, and look for "items" / "results" - some sort of repeated array of the result resources. A "get" API shouldn't have that.

We could talk about this in the future too. For now singletons aren't that common, so I'd be fine if this was a follow-up issue.

Reply:
If you can file an issue, I'm fine with merging this PR as-is, with the check specifically for the get with a path element at the end.